samples/debug/ppi_trace/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _ppi_trace_sample:
+
+PPI trace
+#########
+
+The PPI trace sample shows how to trace hardware events on GPIO pins.
+It uses the :ref:`ppi_trace` module.
+
+Overview
+********
+
+The sample initializes trace pins to observe the following hardware events:
+
+* RTC Compare event (``NRF_RTC_EVENT_COMPARE_1``)
```

Comment:
Are these event names (all three) correct? You had "RTC_COMPARE_1", but I couldn't find that anywhere. What does it refer to?

Reply:
yes, it should be like you did, with hal names.